pootle/apps/pootle_store/util.py

```diff
@@ -36,6 +38,12 @@
     FUZZY: "fuzzy",
     TRANSLATED: "translated"}
 
+TTK_STATES = {
+    pounit.S_FUZZY: FUZZY,
+    pounit.S_OBSOLETE: OBSOLETE,
+    pounit.S_TRANSLATED: TRANSLATED,
+    pounit.S_UNTRANSLATED: UNTRANSLATED}
```

Comment:
This is PO centric. How this will affect projects in another formats, for example XLIFF?